shanoir-ng-front/src/app/acquisition-equipments/acquisition-equipment/acquisition-equipment.component.ts

```diff
@@ -112,6 +115,10 @@ export class AcquisitionEquipmentComponent extends EntityComponent<AcquisitionEq
     }
 
     buildForm(): UntypedFormGroup {
+        if (this.fromImport) {
```

Comment:
I think It doesn't belong here, buildForm is for building the form object.

Imo it would be better in initCreate() or even in prefill() which is called by initCreate()